docs/technical-documentation/operator-credential-creation/intial-credential-setup.md

```diff
@@ -0,0 +1,59 @@
+# Initial Credential Setup
```

Comment:
I suggest to add somewhere a warning regarding the naming of the application on wallet provider side, that it must be ensured that the application name is "catena-x-portal" https://github.com/eclipse-tractusx/ssi-credential-issuer/blob/v1.0.0/src/externalservices/Wallet.Service/Services/WalletService.cs#L48

And please raise an enhancement issue to make this value as well as the /api/v2.0.0/credentials path configurable

Reply:
the documentation is adjusted.

Enhancement issue is created: https://github.com/eclipse-tractusx/ssi-credential-issuer/issues/226